assets/js/modules/adsense/dashboard/dashboard-widget.js

```diff
@@ -223,11 +224,11 @@ class AdSenseDashboardWidget extends Component {
 									/>
 								</Layout>
 							</div>
-							<div className={ `
+							<div className={ classnames( `
 								mdc-layout-grid__cell
-								mdc-layout-grid__cell--span-12
-								${ wrapperClass }
-							` } >
+								mdc-layout-grid__cell--span-12`,
+							wrapperClass
```

Comment:
This indentation seems off; if the linter is cool with it we should tweak things as this doesn't seem right.